pkg/specgen/generate/kube/kube.go

```diff
@@ -289,6 +291,14 @@ func ToSpecGen(ctx context.Context, opts *CtrSpecGenOptions) (*specgen.SpecGener
 		volume.MountPath = dest
 		switch volumeSource.Type {
 		case KubeVolumeTypeBindMount:
+			// If the container has bind mounts, we need to check if
+			// a selinux mount option exists for it
+			for k, v := range opts.Annotations {
+				// Make sure the z/Z option is not already there (from editing the YAML)
+				if k == volumeSource.Source && !util.StringInSlice("z", options) && !util.StringInSlice("Z", options) {
```

Comment:
I believe, but I'm just checking, `z` and `Z` are unique values in options.  I.e. there's no chance one of them would be part of another string value in options right?

Reply:
there are unique